plugins/SLURM-Integration/install.sh

```diff
@@ -26,20 +26,85 @@ INSTALLATION_FAILED='Installation failed'
 install_status=0
 WORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+# Get the current PATH
```

Comment:
just need to do
```export PATH=$PATH:/some/new/path1:/new/path2```

and let ```type``` search through PATH for a binary
lines 30-38, 43-72 are not needed